frontend/src/components/AccessGame/Deck.vue

```diff
@@ -1,5 +1,8 @@
 <template>
-  <div v-if="player && player.id !== currentRound.judgeId && currentRound.status === 'created'" class="deck my-5">
+  <div
+    v-if="player && currentRound && player.id !== currentRound.judgeId && currentRound.status === 'created'"
```

Comment:
Since we're already refactoring: I'd put this in a computed property, but I can do that later as well.

Reply:
```suggestion
    v-if="player?.id !== currentRound?.judgeId && currentRound?.status === 'created'"
```

Reply:
Nope, not supported in vue :(